lib/gems/pending/encryption/CryptString.rb

```diff
@@ -1,34 +0,0 @@
-require 'openssl'
-
-class CryptString
```

Comment:
Yay!

nice to get this out of here.

Can we drop this functionality all together instead of merging into `MiqPassword::Key`?